crates/core/executor/src/events/alu.rs

```diff
@@ -27,15 +27,16 @@ pub struct AluEvent {
     /// The third operand.
     pub c: u32,
     /// The result of the operation.
-    pub sub_lookups: [u128; 6],
+    /// make in the format of [``LookupId``; 6]
```

Comment:
comments should be full sentence with proper punctuation, this is unprofessional